_guides/getting_started.md

```diff
@@ -126,113 +231,155 @@ best constant = 0.500000
+Notice that the `since last` data dropped to `0` on line six, `example 47` (25 passes over 3 examples result in 75 examples). This result implies that, by looking at the same 3 lines of data 25 times, we reached a _perfect predictor_. Not surprising, given that with three examples and five features for each example. 
 
-## Saving your model (a.k.a. regressor) into a file
+>**Note:** You have to add `--holdout_off` when running multiple-passes. VW automatically switches to _over-fit avoidance_ mode by holding 10% of the examples and evaluating performance on the data being held instead of using online training progressive loss algorithms.
 
-By default VW learns the weights of the features and keeps them in an in memory vector. If you want to save the final regressor weights into a file, add `-f filename`:
+**Use** `--holdout_period period` to change the one in 10 default holdout in VW. 
+
+## Saving regressor model files
+
+Vowpal Wabbit learns the weights of the features and keeps them in an _in-memory vector_ by default. 
+
+**Add** `-f filename` to create and save the final regressor file. For example:
 
 ```sh
 vw house_dataset -l 10 -c --passes 25 --holdout_off -f house.model
 ```
 
-## Getting predictions
+## Make predictions with Vowpal Wabbit 
 
-We want to make predictions of course, this can be done by supplying the `-p filename` option. Stdout can be used as below:
+You can make predictions in VW by supplying the `-p filename`. For example, using filename `stdout`:
 
 ```sh
 vw house_dataset -p /dev/stdout --quiet
 ```
 
-The output is:
+The output for this command is:
+
 ```
 0.000000
 0.000000 second_house
 1.000000 third_house
 ```
 
-- The 1st output line `0.000000` is for the 1st example which has an empty tag.
-- The 2nd output `0.000000 second_house` is for the 2nd example. You'll notice the tag appears here. This is the primary use of the tag: mapping predictions to the examples they belong to.
-- The 3rd output `1.000000 third_house` is for the 3rd example. Clearly, some learning happened, because the prediction is now `1.000000` while the initial prediction was set to `0.5`.
+The first line, `0.000000` refers to the first example, which has an empty tag.
+The second line, `0.000000 second_house` refers to the second example. Notice that the tag appears here. The primary use of the tag is mapping predictions to the corresponding examples.
+The third output, `1.000000 third_house` refers to the third example. The initial prediction was set to `0.5`, and the prediction is now `1.000000`. This result means that _some_ learning occurred. 
+
+In the last example, the model was built in memory incrementally, as VW analyzed the examples. In other words, you predicted _while you learned_.
```

Comment:
analyzed has no meaning here.

Reply:
Is there a better, more accurate term to replace "went over?"